client/backoff_test.go

```diff
@@ -2,13 +2,12 @@ package client
 
 import (
 	"context"
-	"math"
 	"testing"
 	"time"
 )
 
 func TestBackoff(t *testing.T) {
-	delta := time.Duration(0)
+	results := []time.Duration{0 * time.Second, 100 * time.Millisecond, 600 * time.Millisecond, 1900 * time.Millisecond, 4300 * time.Millisecond, 7900 * time.Millisecond}
```

Comment:
Can you put this on multiple lines so its readable?

Reply:
Of course. A few minutes, please.

Reply:
87753ad28953cc1635f734edaf61006f18442fe2